app/main/views/feedback.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+from werkzeug.exceptions import ServiceUnavailable
+from werkzeug.datastructures import MultiDict
+from werkzeug.urls import url_parse
+
+from flask import current_app, request, redirect, flash, Markup
+
+from .. import main
+
+
+@main.route('/feedback', methods=["POST"])
+def send_feedback():
+    feedback_config = current_app.config['DM_FEEDBACK_FORM']
+    form_data = MultiDict()
+    for field, google_form_field in feedback_config['fields'].items():
+        form_data.setlist(google_form_field, request.form.getlist(field))
+
+    result = requests.post(feedback_config['uri'], list(form_data.items(multi=True)))
+    if result.status_code != 200:
+        current_app.logger.error(
+            "Feedback form submission error - unexpected response {}.".format(result.status_code))
+        raise ServiceUnavailable('There was a problem submitting your feedback. Please try again later.')
+
+    came_from = url_parse(request.form['uri'])
+    # strip netloc and scheme as we should ignore attempts to make us redirect elsewhere
+    replaced = came_from._replace(scheme='', netloc='')
+
+    flash(Markup(
+        """Thank you for your message. If you have more detailed feedback, please email
+        <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or
+        <a href="https://airtable.com/shrkFM8L6Wfenzn5Q">take part in our research</a>.
```

Comment:
```
https://airtable.com/shrkFM8L6Wfenzn5Q
```
Not worth making this configurable?

Reply:
I reckon when it changes the content here will change too. Certainly the content should at least be reviewed if the URL changes, so having them in the same spot seems ok to me.